helper-cli/src/main/kotlin/commands/SetDependencyRepresentationCommand.kt

```diff
@@ -64,6 +64,11 @@ class SetDependencyRepresentationCommand : CliktCommand(
         GRAPH {
             override fun convert(result: AnalyzerResult): AnalyzerResult =
                 DependencyGraphConverter.convert(result)
+        },
+
+        /** The classic dependency tree format. */
+        TREE {
+            override fun convert(result: AnalyzerResult): AnalyzerResult = result.withScopesResolved()
```

Comment:
Uhm, do we have both `OrtResult.withResolvedScopes()` and `AnalyzerResult.withScopesResolved()` functions in ORT? If so, can we align this by add a commit to this PR? Personally, I'd prefer the "withResolvedScopes" name.